app/models/custom_actions/actions/strategies/custom_field.rb

```diff
@@ -30,7 +30,10 @@
 
 module CustomActions::Actions::Strategies::CustomField
   def apply(work_package)
-    work_package.send(custom_field.attribute_setter, values) if work_package.respond_to?(custom_field.attribute_setter)
+    if work_package.respond_to?(custom_field.attribute_setter)
+      set_custom_field_value(work_package)
+      validate_custom_field(work_package)
+    end
```

Comment:
These lines are repeated a couple of times in the application. Maybe a chance for extracting them into a module.

Reply:
Will be addressed separately.